src/pages/index.tsx

```diff
@@ -1,5 +0,0 @@
-import { AllOverview, route as overviewRoute } from "./overview";
-
-export { AllOverview }
-
-export const routes = [overviewRoute];
```

Comment:
Can you leave this file in place and add a comment why we decided against using it on purpose